README.md

```diff
@@ -180,10 +180,10 @@ If the default ingress gateway setup does not fit your need, you can choose to s
 - [Configure Custom Domain](https://knative.dev/docs/serving/using-a-custom-domain/)
 - [Configure HTTPS Connection](https://knative.dev/docs/serving/using-a-tls-cert/)
 
-### Setup Monitoring
-- [Metrics](https://knative.dev/docs/serving/accessing-metrics/)
+### Metrics, Tracing and Debugging
+- [AutoScaling Metrics](https://knative.dev/docs/serving/autoscaling/autoscaling-metrics/)
```

Comment:
hmm these metrics are very useful, not sure why they deprecated.

Reply:
Yeah, agree. I use the observability plugin stuff myself. Looks like theres [open discussion](https://github.com/knative/docs/issues/2931) about the future of metrics. Let me see if I can piece together some guidance around metrics, post-observability, in this pr to start with.

Reply:
Looks like the [KFSERVING_DEBUG_GUIDE](https://github.com/kubeflow/kfserving/blob/master/docs/KFSERVING_DEBUG_GUIDE.md#investigate-performance-issues) needs updating as well

Reply:
I think those knative metrics are still exposed, is that the case they drop the support for those dashboards and suggest user building those on their own?

Reply:
There is more in depth information in this [Knative doc](https://github.com/knative/serving/tree/master/vendor/knative.dev/pkg/metrics#the-plan). From the looks of it, "The Plan" has changed.